get-pr-info/action.yml

```diff
@@ -0,0 +1,89 @@
+name: Get information about a pull request
+description: >
+  This action provides information about a pull request,
+  analogous to `github.pull_request` context in workflows,
+  triggered by `pull_request` events
+
+inputs:
+  owner:
+    description: Repository owner
+    required: true
+  repo:
+    description: Repository name
+    required: true
+  pr_number:
+    description: Pull request number
+    required: true
+  token:
+    description: GitHub personal access token (PAT).
+    required: false
+    default: ${{ github.token }}
+  set-env:
+    description: Set environment variables
+    required: false
+    default: 'true'
+
+outputs:
+  base_repo:
+    description: Owner/name of the main repository, where the pull request is opened
+    value: ${{ fromJSON(steps.get-info.outputs.result).base_repo }}
+  base_ref:
+    description: The target branch of the pull request
+    value: ${{ fromJSON(steps.get-info.outputs.result).base_ref }}
+  base_sha:
+    description: The target branch HEAD's SHA
+    value: ${{ fromJSON(steps.get-info.outputs.result).base_sha }}
+  head_repo:
+    description: Owner/name of repository with the feature branch
```

Comment:
I am not a native speaker, but `the repository` sounds better for me.

Reply:
You are right.